qokit/qaoa_objective.py

```diff
@@ -84,9 +86,11 @@ def g(gamma, beta):
         backend = Aer.get_backend("statevector_simulator")
 
         def g(gamma, beta):
-            qc = parameterized_circuit.bind_parameters(np.hstack([np.asarray(beta) / 2, np.asarray(gamma) / 2]))
-            result = execute(qc, backend).result()
-            sv = reverse_array_index_bit_order(result.get_statevector())
+            qc = parameterized_circuit.assign_parameters(np.hstack([np.asarray(beta) / 2, np.asarray(gamma) / 2]))
+            # qc = parameterized_circuit.bind_parameters(np.hstack([np.asarray(beta) / 2, np.asarray(gamma) / 2]))
```

Comment:
Can you please remove commented out code?

Reply:
Done!